Week1/homework/Masoud/js-exercises/EX5-roundAAumberAndLogIt.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let greater;
+if (a >= z)
+{
```

Comment:
Whatever you use above make sure they are all consitent. Generally I prefer this format:
```
if (condition) {
...
}
```